images/win/scripts/Tests/Java.Tests.ps1

```diff
@@ -1,5 +1,5 @@
 Describe "Java" {
-    [array]$jdkVersions = (Get-ToolsetContent).java.versions | ForEach-Object { @{Version = $_} }
+    [array]$jdkVersions = (Get-ToolsetContent).java.vendors.versions | Sort | Get-Unique | ForEach-Object { @{Version = $_} }
```

Comment:
I think we need only versions from default vendor here. Otherwise, `It "Java <Version>" -TestCases $jdkVersions {` test won't work properly.

Reply:
Done